containers/podman-desktop
# chore: use git hooks for linter/formatter and signed-off/conventional-commits

### What does this PR do?
Add a pre-commit hook for linter/formatter
Add commit-msg hook for checking conventional commit and adding Signed-Off

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

fixes https://github.com/containers/podman-desktop/issues/2845


### How to test this PR?

Change files (with unformatted style or lint errors)
Try to add the files to staging area and do a commit
it should fail
